app/controllers/planning.rb

```diff
@@ -0,0 +1,107 @@
+get '/mission/new' do
+  #this should only provide currently avaliable ppl,
+  #off-line until the 'prevent double booking' validation is added
+  @personnel = Soldier.all
+  #as w/ ppl
+  @trucks = Truck.all
+  @trailers = Trailer.all
+  erb :'planning/new_mission'
+end
+
+
+post '/mission' do
```

Comment:
there's way too much going on in this method. keep methods single responsibility :+1: